### PR TITLE
Update documents for 1.3 release

### PR DESCRIPTION
**Pull request by agnieszka-m**

### CODE REVIEW

File: docs/latest/components/joinAnswersmdx.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- You use a different reader for each document type and you get predicted answers form each reader separately
+- You use the `JoinAnswers` node to join the answers in a single list 
+
+Here's what an example pipeline with the `JoinAnswers` node could look like:
+![image](/img/JoinAnswers_pipeline.jpg)
+
+## Usage
+
+When initializing this node, you must specify the join mode that you want to use. There are two modes available:
+- `concatenate` Combines documnets from multiple readers
```
Comment (bogdankostic) [inline docs/latest/components/joinAnswersmdx.mdx]: typo

File: docs/latest/guides/evaluation.mdx [hunk 2]
```diff
@@ -47,6 +47,39 @@ When using `pipeline.eval()` in either **integrated** or **isolated** modes,
 Haystack evaluates the correctness of an extracted answer by looking for a match or overlap between the answer and prediction strings.
 Even if the predicted answer is extracted from a different position than the correct answer, that's fine, as long as the strings match.
 
+## Evaluating Information Retrieval with BEIR
+
+Haystack has an integration with the Benchmarking Information Retrieval (BEIR) tool that you can use to assess the quality of your information retrieval pipelines. BEIR contains preprocessed datasets for evaluating retrieval models in 17 languages.
+
+To use BEIR, install Haystack with BEIR dependency:
+```python
```
Comment (bogdankostic) [inline docs/latest/guides/evaluation.mdx]: I would remove the python tag here, as this is a shell command.

**Review by bogdankostic — COMMENTED** [review]
Looking good, I like the explanation of the new nodes and BEIR integration!
I would rename the file `joinAnswersmdx.mdx` to `join_answers.mdx` and the file `routeDocumentsmdx.mdx` to `route_documents.mdx` to have a consistent naming of the files. (I think this also fixes the `404 - There's no doc for the page you requested.` error that we currently see when trying to access the RouteDocuments or JoinAnswers page from the vercel deployment).

File: docs/latest/components/joinAnswersmdx.mdx (hunk 1 above)
Comment (agnieszka-m) [reply docs/latest/components/joinAnswersmdx.mdx]: fixed

File: docs/latest/guides/evaluation.mdx (hunk 2 above)
Comment (agnieszka-m) [reply docs/latest/guides/evaluation.mdx]: done

**Review by PiffPaffM — APPROVED** [review]
@agnieszka-m Look good to me! I changed two minor things. You can find the test deployment here, if you want to check it before you merge: [haystack-website-a5zqkx31m-deepset-overnice.vercel.app](https://haystack-website-a5zqkx31m-deepset-overnice.vercel.app/) (I could not find any problems after checking some pages.)

**Comment by agnieszka-m** [comment]
Thanks Markus!

**Pull request merged**